open-metadata-implementation/view-services/glossary-author-view/glossary-author-view-server/src/main/java/org/odpi/openmetadata/viewservices/glossaryauthor/services/GlossaryAuthorViewGlossaryRESTServices.java

```diff
@@ -390,10 +390,12 @@ public GlossaryAuthorViewGlossaryRESTServices() {
      */
     public SubjectAreaOMASAPIResponse<SubjectAreaOMASAPIResponse<Term>> createMultipleTermsInAGlossary(String serverName, String userId, String guid, Term[] terms) {
         final String methodName = "createMultipleTermsInAGlossary";
+        RESTCallToken token = restCallLogger.logRESTCall(serverName, userId, methodName);
+
         List<SubjectAreaOMASAPIResponse<Term>> termList = new ArrayList<>();
         SubjectAreaOMASAPIResponse<SubjectAreaOMASAPIResponse<Term>> response = new SubjectAreaOMASAPIResponse<>();
+        AuditLog auditLog = null;
```

Comment:
AuditLog is not initialized. You need this call -> `auditLog = instanceHandler.getAuditLog(userId, serverName, methodName);` in `try catch` construction.

Reply:
:-) Good catch @immmus

Reply:
changed